opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "&lt;nil&gt;"
```

Comment:
@Zimmi48 can we define some default email for Coq-community OPAM script maintenance (`maintainer` field)? Or should it be the package maintainer (who may not want to make their email public)? In any case, it's clear that this field should not be HTML processed.

Reply:
Indeed, well spotted. I don't know the package format so well: does it have to be an e-mail and do we have to provide this field? Would you be willing to be the "default" maintainer or should we rather create a coq-community-maintainers mailing list?

Reply:
According to the OPAM spec, the maintainer field is required. However, as far as I understand it, there is nothing that forces the maintainer field to be an e-mail address, just a convention used in most OPAM repositories. I can be the default maintainer for now.